ddtrace/bootstrap/sitecustomize.py

```diff
@@ -99,11 +99,7 @@ def update_patched_modules():
     if priority_sampling:
         opts["priority_sampling"] = asbool(priority_sampling)
 
-    # FIXME: Remove as part of the deprecation of collect_metrics
-    opts["collect_metrics"] = asbool(get_env("runtime_metrics", "enabled"))
-
-    if opts:
```

Comment:
Probably still worth having this line? Else `configure()` will be called needlessly (even though it _shouldn't_ do anything)

Reply:
I had to remove this because a test that was checking for the log output generated by this call was failing. This showed that `configure` got called all the time anyway because `collect_metrics` was always set to either `True` or `False`.

Reply:
ah true :+1: